src/database-sqlite3.cpp

```diff
@@ -56,6 +65,50 @@ SQLite format specification:
 			 sqlite3_errmsg(m_database)); \
 	}
 
+static int sqlite3BusyHandler(void *data, int count)
+{
+	s64 &first_time = reinterpret_cast<s64 *>(data)[0];
+	s64 &prev_time = reinterpret_cast<s64 *>(data)[1];
+	s64 cur_time = getTimeMs();
+
+	if (count == 0)
+		first_time = cur_time;
```

Comment:
It can't be used before it is set. I'll make that more explicit to avoid any possible confusion.